src/main/java/fr/lunatech/timekeeper/services/UserEventService.java

```diff
@@ -35,6 +37,28 @@
         return getEventsByUser(ownerId, monthNumber, year, TimeKeeperDateUtils::getMonthNumberFromDate);
     }
 
+    /**
+     * @param id
+     * @param context TODO SECURE ME WITH a CAN ACCESS
```

Comment:
Can you add a ticket on JIRA to track this task? Thanks

Reply:
TK-472